Makefile

```diff
@@ -352,7 +352,7 @@ ifeq (, $(shell command -v opa))
 	@{ \
 	set -e ;\
 	mkdir -p ${HOME}/.local/bin ; \
-	curl -sL -o ${HOME}/.local/bin/opa https://openpolicyagent.org/downloads/v0.57.1/opa_linux_amd64_static ; \
+	curl -sL -o ${HOME}/.local/bin/opa https://openpolicyagent.org/downloads/v0.62.1/opa_linux_amd64_static ; \
```

Comment:
that's unrelated to the SDK, right? I also wonder, why 0.62.1 while there's 0.65.0?
I'd suggest to separate this out to a different PR

Reply:
yeah that just alignment with bazel where we use 0.62...i'll make another PR for OPA